src/editors/_baseEditor.js

```diff
@@ -371,20 +371,28 @@ class BaseEditor {
    *
    * @returns {string}
    */
-  getEditedCellsZIndex() {
+  getEditedCellsZIndexClass() {
```

Comment:
This method is not used anywhere? Then we should remove unused code.

I'm not sure if the classes here are correct. They would give the editor same `z-index` as overlay.

Reply:
For now, this method isn't used because we decided to permanent `z-index` for `editor` in every overlay.
We will use it in the next step when we implement separate `editor UI` for `editor input`. This method has only one goal - return appropriate overlay sections to the editor. And then we are going to add `4` to base `z-index`.